model/task_json_api.go

```diff
@@ -0,0 +1,139 @@
+package model
+
+import (
+	"time"
+
+	"github.com/evergreen-ci/evergreen/db"
+	"github.com/evergreen-ci/evergreen/model/version"
+	mgo "gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+)
+
+// CommitInfo represents the information about the commit
+// associated with the version of a given project. This is displayed
+// at the top of each header for each project.
+type PerfCommitInfo struct {
```

Comment:
I think we should keep these names like they were before. Task json can be used for any task, not just perf

Reply:
but it isn't.